lib/kino_flame/runner_cell.ex

```diff
@@ -126,6 +222,33 @@ defmodule KinoFLAME.RunnerCell do
 
     env = {:%{}, [], envs}
 
+    backend_ast =
+      quote do
+        {FLAME.FlyBackend,
+         [
+           unquote_splicing(specs_opts),
+           env: unquote(env)
+         ]}
+      end
+
+    to_quoted_pool(attrs, backend_ast)
+  end
+
+  defp to_quoted(%{"backend" => "k8s"} = attrs) do
+    multiline_runner_pod_template =
+      {:sigil_y, [delimiter: ~S["""]], [{:<<>>, [], [attrs["runner_pod_template"]]}, []]}
```

Comment:
```suggestion
      {:sigil_y, [delimiter: ~S["""]], [{:<<>>, [], [attrs["runner_pod_template"] <> "\n"]}, []]}
```

We need a trailing newline, otherwise the closing `"""` will stick to it's previous line.

Reply:
but we will always add a \n even if it is already there... how about this?

```ex
    k8s_pod_template = attrs["k8s_pod_template"] |> String.trim_trailing("\n")

    multiline_k8s_pod_template =
      {:sigil_y, [delimiter: ~S["""]], [{:<<>>, [], [k8s_pod_template <> "\n"]}, []]}
```

Reply:
It's unlikely to have a trailing newline in cells. Regardless adding the newline does change the actual string. But, once the yaml is parsed, it's going to be the same, so it should be fine?

Reply:
ok. let me remove the newline in the `@default_pod_template` and then we skip the trimming.

Reply:
committed.

Reply:
Good call about the template!